src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/ColorFieldTransformer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import log from 'loglevel';
+import type {Node} from 'react';
+import type {FieldTransformer} from '../types';
+import colorFieldTransformerStyles from './colorFieldTransformer.scss';
+
+export default class ColorFieldTransformer implements FieldTransformer {
+    transform(value: *): Node {
+        //test si contain # et 3 ou 6 charateres
+        if (!/^#[0-9a-f]{3}([0-9a-f]{3})?$/i.test(value)) {
+            log.error(`Transformer parameter "${value}" needs to be of type hexadecimal color.`);
+            return value;
```

Comment:
would return `null` here to be consistent with the `NumberFieldTransformer` and `DateFieldTransformer`
```suggestion

            return null;
```